table/builder_test.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package table
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/dgraph-io/badger/options"
+	"github.com/dgraph-io/badger/y"
+)
+
+func TestTableIndex(t *testing.T) {
```

Comment:
Does it make sense to add test for old format?

Reply:
I don't think so. This PR replaces the old format with a new one. If I add tests for the old one, the build won't succeed ;)